src/sentry/deletions/defaults/apigrant.py

```diff
@@ -0,0 +1,13 @@
+from ..base import ModelDeletionTask
+
+
+class ModelApiGrantDeletionTask(ModelDeletionTask):
+    """
+    Normally ApiGrants are deleted in bulk, but for cascades originating from sentry app installation, we wish to use
```

Comment:
This pattern is repeated identically across a few files here. Should this just be a base class? I see that it adds explicitness for the model type. But in what cases would you extend the behavior further?

Reply:
I think one case you might extend it is if in the future you did add a `status` tracking asynchronous deletion.  Right now, I kept the synchronous nature of the delete (deletions.exec_sync) but in the future, if I were to make this a job, I'd want this model to pre-emptively mark itself for async deletion so that it stops showing up or being available even before the deletion work itself completes.